tidb-binlog/node/node.go

```diff
@@ -0,0 +1,73 @@
+package node
+
+import (
+	"time"
+)
+
+var (
+	// DefaultRootPath is the root path of the keys stored in etcd, the `2.1` is the tidb-binlog's version.
+	DefaultRootPath = "/tidb-binlog/2.1"
```

Comment:
why `2.1`? lol

Reply:
we will publish the new version binlog with tidb 2.1.

Reply:
we only have v1 now :sob: